ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -121,12 +122,19 @@ def get_action(self, brain_info: BrainInfo) -> ActionInfo:
         to be passed to add experiences
         """
         if len(brain_info.agents) == 0:
-            return ActionInfo([], [], [], None, None)
-
+            return ActionInfo([], [], [], None)
+
+        self.remove_memories(
+            [
```

Comment:
If you wanted to be a bit more pythonic, I think you could do
```
[
  agent 
  for agent, done in zip(brain_info.agents, brain_info.local_done)
  if done
]
```
but no worries if you like the current way better.